go/vt/sqlparser/truncate_query.go

```diff
@@ -17,17 +17,28 @@ limitations under the License.
 package sqlparser
 
 import (
-	"flag"
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
 )
 
 var (
 	// TruncateUILen truncate queries in debug UIs to the given length. 0 means unlimited.
-	TruncateUILen = flag.Int("sql-max-length-ui", 512, "truncate queries in debug UIs to the given length (default 512)")
+	TruncateUILen int
```

Comment:
if the default was not the zero-value, you should assign it here:

```go
TruncateUILen = 512
```

also do these need to be exported, or can we do a similar thing to https://github.com/vitessio/vitess/pull/11010

Reply:
I have done the default change and unexported the variables, albeit only using Getter and Setter functions. Is there some other benefit we get by doing it in the way of flag parsing function registeration?

Reply:
> Is there some other benefit we get by doing it in the way of flag parsing function registeration?

I don't quite follow this question, are you talking about the default or the un-exporting?

Reply:
un-exporting. I have done it using just a Getter and Sette function. Is that alright?